.github/workflows/testing.yml

```diff
@@ -9,16 +9,16 @@ jobs:
     name: Drupal ${{ matrix.drupal-core }} (PHP ${{ matrix.php-versions }})
     runs-on: ubuntu-20.04
     env:
-      extensions: mbstring, xml, pdo_sqlite, gd, opcache
+      extensions: :sodium, Imagick, gd, PDO, mysql, pdo_mysql, mbstring, xmlwriter, pdo, date, dom, filter, hash, json, pcre, session, SimpleXML, SPL, tokenizer, xml, curl
     strategy:
       fail-fast: false
       matrix:
-        php-versions: ['7.3', '7.4', '8.0', '8.1']
+        php-versions: ['7.4', '8.0', '8.1']
         drupal-core: ['9.3.x']
         include:
-          # Extra run to also test on latest Drupal 8 and PHP 7.2.
-          - php-versions: '7.2'
-            drupal-core: '8.9.x'
+          # Extra run to also test on latest Drupal 10 and PHP 8.1.
+          - php-versions: '8.1'
+            drupal-core: '10.0.x'
```

Comment:
not sure we should enable Drupal 10 testing since it is not even in beta yet. That could lead to unrelated test fails in future pull requests. I think we can risk it here and if we see Drupal 10 failing for unknown reasons before a stable release we can throw out Drupal 10 testing again.

Reply:
You could also mark it as "Allowed to fail" for now

Reply:
Great point, that is exactly what we should do. Is that possible with Github actions and can you set that in this PR?

Reply:
You just don't set the D10 job to "required" in your repo settings, I think

Reply:
OK, I think that could be enough.